### PR TITLE
New version: JohnMacFarlane.Pandoc version 3.6

### PR DESCRIPTION
**Pull request by damn-good-b0t**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/197015)

### CODE REVIEW

File: manifests/j/JohnMacFarlane/Pandoc/3.6/JohnMacFarlane.Pandoc.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ReleaseDate: 2024-12-08
+AppsAndFeaturesEntries:
+- DisplayName: Pandoc 3.6
+  Publisher: John MacFarlane
+  ProductCode: '{9AF3E035-C175-4C5E-B71A-4EA4F890B4F7}'
+  UpgradeCode: '{A68E8EF6-ABB1-4F22-A3C5-68DFDF0AB562}'
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%/Pandoc'
+Installers:
+- Architecture: x64
```
Comment (DuckDuckStudio) [inline manifests/j/JohnMacFarlane/Pandoc/3.6/JohnMacFarlane.Pandoc.installer.yaml:24]:
```suggestion
- Architecture: x64
  Scope: user
```

File: manifests/j/JohnMacFarlane/Pandoc/3.6/JohnMacFarlane.Pandoc.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  InstallerUrl: https://github.com/jgm/pandoc/releases/download/3.6/pandoc-3.6-windows-x86_64.msi
+  InstallerSha256: EFC621BB08901FBF9C78CC71D5C00D8E67EB9318F8EF994D908D71E0D28EB702
+  InstallerSwitches:
+    Custom: ALLUSERS=1
+- Architecture: x64
```
Comment (DuckDuckStudio) [inline manifests/j/JohnMacFarlane/Pandoc/3.6/JohnMacFarlane.Pandoc.installer.yaml:29]:
```suggestion
- Architecture: x64
  Scope: machine
```

**Review by DuckDuckStudio — COMMENTED** [review]
(According to the previous version of the manifest.)

File: manifests/j/JohnMacFarlane/Pandoc/3.6/JohnMacFarlane.Pandoc.installer.yaml (hunk 1 above)
Comment (a-mnich) [reply manifests/j/JohnMacFarlane/Pandoc/3.6/JohnMacFarlane.Pandoc.installer.yaml:23]: fixed with https://github.com/microsoft/winget-pkgs/pull/197015/commits/9c1ce25174ef6e68d4f1f6368a8d3866315f1a71

File: manifests/j/JohnMacFarlane/Pandoc/3.6/JohnMacFarlane.Pandoc.installer.yaml (hunk 2 above)
Comment (a-mnich) [reply manifests/j/JohnMacFarlane/Pandoc/3.6/JohnMacFarlane.Pandoc.installer.yaml:29]: fixed with https://github.com/microsoft/winget-pkgs/pull/197015/commits/9c1ce25174ef6e68d4f1f6368a8d3866315f1a71

**Review by ItzLevvie — APPROVED** [review]

File: manifests/j/JohnMacFarlane/Pandoc/3.6/JohnMacFarlane.Pandoc.installer.yaml (hunk 1 above)
Comment (DuckDuckStudio) [reply manifests/j/JohnMacFarlane/Pandoc/3.6/JohnMacFarlane.Pandoc.installer.yaml:23]: This pull request still requires you to mark the conversation as resolved. 🙏

**Pull request merged**